lib/tasks/pirce_groups.rake

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+namespace :price_group do
+  # This creates a new global price group, if it does not exist, and adds
+  # a PriceGroupDiscount for it to every ScheduleRule. If the price group
+  # already exists, this will just add PriceGroupDiscounts for it to ScheduleRules
+  # that are missing then.
+  #
+  # rake price_group:add_global_price_group["new group",true,false,15]
+  desc "Create a new global price group and/or setup schedule rules for it"
+  task :add_global_price_group, [:name, :internal, :admin_editable, :discount] => :environment do |_t, args|
+    name = args[:name]
+    is_internal = args[:internal].casecmp("true").zero?
```

Comment:
Not a blocking comment, but curious why you opted for `casecmp` instead of `== "true"` here

Reply:
@giladshanan this was a RuboCop suggestion https://docs.rubocop.org/rubocop-performance/cops_performance.html#performancecasecmp